### ISSUE TITLE
Rescue Bill Williamson

### ISSUE THREAD

**Issue by t0mtee** [subvod/rdr2-maps#1]
Hi, just discovered this and it looks very interesting. You state that the 'Rescue Bill Williamson' event is only available in Chapter 2, while the RDR2 wiki says it's available through Chapters 2-4. I've also seen a decent amount of stories online that say they've had the event in Chapter 3, and a few (not a lot) say they've had it in Chapter 4. Maybe some more research into this encounter would be good? It's the only missable random encounter and is a pain to try and randomly trigger. Thanks.

**Comment by subvod** [comment]
I haven't personally verified everything listed in the doc. Most information was taken from online docs and forum posts. Reference links are provided where applicable. One difference I've noticed from the existing notes on missable encounters is that I've not been able to pop Dutch's pipe request in Chapter 2; only earliest I've gotten it is in Chapter 3. That was noted recently.

Otherwise, if you run across proof from anyone else that something can be done/obtained elsewhere, feel free to link it. This will remain a WIP project, given how massive this game is. Even to me, still finding new things once in a while despite having played several thousand hours since 2020.

Back on researching and what not, like I said feel free to link proof if you come across it. I'll note it in the doc and link the original author where necessary. As of recent, I've less time to be on RDR2, sparing the circumstances. That being the case, contributions from others are welcome. :)

**Comment by t0mtee** [comment]
Ok, cool. This does seem to be the one of the most comprehensive list of missables I've seen, and trust me, I've searched a lot. Here is the list of missables I've compiled, you may be missing a couple (I noticed the Closed Winter Gunslinger isn't on yours, you may want to add that on)
[Red Dead Redemption 2.xlsx](https://github.com/subvod/rdr2-maps/files/8984859/Red.Dead.Redemption.2.xlsx)

**Comment by subvod** [comment]
For missable hats, those are listed [here](https://redd.it/ojj4vc). Though, since I'm on PC, I just use RampageTrainer to add them to my inventory. Even those hats which aren't normally accessible like Plainsman. For documents, [here's a reference](https://redd.it/gh5saj). The comic books are also noted in there, as well.

Some other things are listed [on the Fandom](https://reddead.fandom.com/wiki/Missable_Content_in_Redemption_2). There's also a ton of comprehensive references for missables scattered around mostly Reddit. I did my best to consolidate everything in some orderly fashion when I merged my `RDR2.MD` with the `README.MD` on this project. Maintaining table structure, and what not. Some information is redundant, listed two or three times, and I'm skimming through each day removing that excess info.

As for the closed winter coat, iirc that can only be obtained via glitch and it's not stored in the wardrobe if you remove it. I don't know the exact glitch to get it, but I'll take a look into it. If you run across anything I've not listed, such as those you mentioned prior, let me know and I'll add it in.
